### DISCUSSION TITLE
MountVolume.SetUp failed for volume for GS-engine

### DISCUSSION THREAD

**Discussion by DanielaCasilli** [alibaba/GraphScope#456] [Q&A] [answered]
Hi,

I am trying to get a local version of Graphscope deployed on my Mac and have run into an error when the gs-engine is trying to spin up:

> 2021-06-30 18:20:55,321 [INFO][cluster:643]: [gs-engine-oecfbr-xdc4c]: MountVolume.SetUp failed for volume "data" : hostPath type check failed: /Users/danielacasilli/test_data/ is not a directory
> 2021-06-30 18:22:56,279 [INFO][cluster:643]: [gs-engine-oecfbr-xdc4c]: Unable to attach or mount volumes: unmounted volumes=[data], unattached volumes=[data default-token-2jhtw vineyard-ipc-volume host-shm]: timed out waiting for the condition

The config I am using to create a graphscope session is:
```
k8s_volumes = {
    "data": {
        "type": "hostPath",
        "field": {
            "path": os.path.expanduser("~/test_data/"),
            "type": "Directory"
        },
        "mounts": {
            "mountPath": "/testingdata"
        }
    }
}



config ={'num_workers': 1,
'log_level': 'info',
'show_log': True,
'k8s_namespace': None,
'k8s_gs_image': 'registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.2.1',
'k8s_etcd_image': 'quay.io/coreos/etcd:v3.4.13',
'k8s_image_pull_policy': 'IfNotPresent',
'k8s_image_pull_secrets': [],
'k8s_gie_graph_manager_image': 'registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.2.1',
'k8s_zookeeper_image': 'zookeeper:3.4.14',
'k8s_coordinator_cpu': 0.1,
'k8s_coordinator_mem': '256Mi',
'k8s_vineyard_cpu': 0.1,
'k8s_vineyard_mem': '256Mi',
'k8s_vineyard_shared_mem': '4Gi',
'k8s_engine_cpu': 0.1,
'k8s_engine_mem': '256Mi',
'k8s_waiting_for_delete': False,
'timeout_seconds': 600}

sess = graphscope.session(k8s_volumes=k8s_volumes,config=config, k8s_service_type="LoadBalancer")
```

Any help would be appreciated. 

I am mainly following the standard documentation and have tried without the k8s_volumes specified however it fails to deploy without it.

**Comment by sighingnow** [comment, marked as answer]
The value `data.field.path` is the _hostpath_ in your kubernetes hosts. With docker-desktop on Mac, you need to first add the mac path `/Users` into the shared directories of the docker.

See also https://forums.docker.com/t/how-to-mount-hostpath-using-docker-for-mac-kubernetes/44083/6

**Comment by sighingnow** [comment]
We have added "how to mount on Mac" to the documentation.

See also: https://graphscope.io/docs/latest/getting_started.html#launching-session

**Reply by DanielaCasilli** [reply]
Thanks for sharing - I tried the suggested but I already had the "/Users" directory listed under docker's shared directories and I tried adding "~/test_data" as well but it doesn't seem to have helped.

I also tried without specifying data.field.type as "Directory" however this results in the below error:

> 2021-07-06 05:18:03,725 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.2.1" already present on machine
> 2021-07-06 05:18:03,727 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Created container vineyard
> 2021-07-06 05:18:03,729 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Started container vineyard
> 2021-07-06 05:18:03,745 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Created container engine
> 2021-07-06 05:18:03,839 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Started container engine
> 2021-07-06 05:18:13,943 [INFO][cluster:643]: [gs-engine-ftojjd-vcx4p]: Readiness probe failed: ls: cannot access /tmp/vineyard_workspace/vineyard.sock: No such file or directory
> 2021-07-06 05:19:23,040 [ERROR][cluster:804]: Error when launching GraphScope on kubernetes cluster: LoadBalancer service type is not supported yet.
> Traceback (most recent call last):
>   File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/__main__.py", line 3, in <module>
>     launch_graphscope()
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/coordinator.py", line 933, in launch_graphscope
>     log_level=args.log_level,
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/coordinator.py", line 126, in __init__
>     raise RuntimeError("Coordinator Launching failed.")
> RuntimeError: Coordinator Launching failed.
> Traceback (most recent call last):
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 606, in start
>     self._waiting_for_services_ready()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 509, in _waiting_for_services_ready
>     if wait_for_deployment_complete(
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/utils.py", line 98, in wait_for_deployment_complete
>     raise K8sError("Deployment {} start failed.".format(name))
> graphscope.framework.errors.K8sError: 'Deployment coordinator-ftojjd start failed.'
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/utils.py", line 128, in wrapper
>     return_value = func(*args, **kwargs)
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/session.py", line 415, in __init__
>     self._proc, self._endpoint = self._connect()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/session.py", line 708, in _connect
>     endpoint = self._k8s_cluster.start()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 613, in start
>     raise K8sError(
> graphscope.framework.errors.K8sError: 'Error when launching Coordinator on kubernetes cluster'

**Reply by sighingnow** [reply]
`"~/test_data"` won't work. You need to set the _**absolute path**_ **in the shared directory** as `data.field.path`.

**Reply by sighingnow** [reply]
@siyuan0322 Would you mind adding a note about the relative/absolute path into our documentation?

**Reply by siyuan0322** [reply]
I think the error should be this line `Error when launching GraphScope on kubernetes cluster: LoadBalancer service type is not supported yet`, @DanielaCasilli May I know the version of your mac and docker-desktop?

**Reply by DanielaCasilli** [reply]
@sighingnow apologies, I should've also mentioned I have the absolute path set as well i both the Share directory and data.field.path.

@siyuan0322 MacOS Big Sur version 11.4 (20F71)
Memory 8 GB 2133 MHz LPDDR3
Processor 3.1 GHz Dual-Core Intel Core i5

Docker desktop v3.3.3
K8s v1.19.7

**Reply by lidongze0629** [reply]
Hi, @DanielaCasilli Could you upgrade GraphScope to 0.5.0 and run it again, thanks.
```shell
pip3 install graphscope==0.5.0 -U
```

**Reply by DanielaCasilli** [reply]
@lidongze0629 does this mean I should update my image versions from 0.2.1 to 0.5.0?
```
kind load docker-image registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0
kind load docker-image registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0
```

**Reply by DanielaCasilli** [reply]
Hi @lidongze0629, unfortunately I seem to have the same error still:

> 
> config ={'num_workers': 1,
> 'log_level': 'info',
> 'show_log': True,
> 'k8s_namespace': None,
> 'k8s_gs_image': 'registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0',
> 'k8s_etcd_image': 'quay.io/coreos/etcd:v3.4.13',
> 'k8s_gie_graph_manager_image': 'registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0',
> 'k8s_zookeeper_image': 'zookeeper:3.4.14',
> 'k8s_waiting_for_delete': False,
> 'timeout_seconds': 600}


> coordinator-wskhmk-6b49b5757b-7q7wd: Successfully assigned gs-njhjsl/coordinator-wskhmk-6b49b5757b-7q7wd to kind-control-plane
> coordinator-wskhmk-6b49b5757b-7q7wd: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0" already present on machine
> coordinator-wskhmk-6b49b5757b-7q7wd: Created container coordinator
> coordinator-wskhmk-6b49b5757b-7q7wd: Started container coordinator
> 2021-07-07 01:40:56,413 [INFO][cluster:684]: Launching GIE graph manager ...
> 2021-07-07 01:40:57,659 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Successfully assigned gs-njhjsl/gs-graphmanager-wskhmk-5fc4dc77d-q266l to kind-control-plane
> 2021-07-07 01:41:00,660 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0" already present on machine
> 2021-07-07 01:41:00,662 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Created container manager
> 2021-07-07 01:41:00,664 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Started container manager
> 2021-07-07 01:41:00,667 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Container image "zookeeper:3.4.14" already present on machine
> 2021-07-07 01:41:00,669 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Created container zookeeper
> 2021-07-07 01:41:00,672 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Started container zookeeper
> 2021-07-07 01:41:03,621 [INFO][cluster:797]: GIE graph manager service is ready.
> 2021-07-07 01:41:03,622 [INFO][cluster:541]: Launching etcd ...
> coordinator-wskhmk-6b49b5757b-7q7wd: Readiness probe failed: dial tcp 10.244.0.11:59524: connect: connection refused
> 2021-07-07 01:41:04,761 [INFO][cluster:807]: Etcd is ready, endpoint is 10.96.55.189:58664
> 2021-07-07 01:41:04,762 [INFO][cluster:431]: Launching GraphScope engines pod ...
> 2021-07-07 01:41:05,218 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Successfully assigned gs-njhjsl/gs-engine-wskhmk-tx5rd to kind-control-plane
> 2021-07-07 01:41:08,379 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0" already present on machine
> 2021-07-07 01:41:08,382 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Created container engine
> 2021-07-07 01:41:08,385 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Started container engine
> 2021-07-07 01:41:08,388 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Created container vineyard
> 2021-07-07 01:41:08,391 [INFO][cluster:864]: [gs-engine-wskhmk-tx5rd]: Started container vineyard
> 2021-07-07 01:42:19,837 [ERROR][cluster:1058]: Error when launching GraphScope on kubernetes cluster: LoadBalancer service type is not supported yet.
> Traceback (most recent call last):
>   File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/__main__.py", line 3, in <module>
>     launch_graphscope()
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/coordinator.py", line 1122, in launch_graphscope
>     log_level=args.log_level,
>   File "/usr/local/lib/python3.6/site-packages/gscoordinator/coordinator.py", line 129, in __init__
>     raise RuntimeError("Coordinator Launching failed.")
> RuntimeError: Coordinator Launching failed.
> 2021-07-07 01:42:25,626 [INFO][cluster:684]: Launching GIE graph manager ...
> 2021-07-07 01:42:26,860 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-4fcjh]: Successfully assigned gs-njhjsl/gs-graphmanager-wskhmk-5fc4dc77d-4fcjh to kind-control-plane
> 2021-07-07 01:42:27,854 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Successfully assigned gs-njhjsl/gs-graphmanager-wskhmk-5fc4dc77d-q266l to kind-control-plane
> 2021-07-07 01:42:27,856 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0" already present on machine
> 2021-07-07 01:42:27,857 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Created container manager
> 2021-07-07 01:42:27,859 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Started container manager
> 2021-07-07 01:42:27,860 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Container image "zookeeper:3.4.14" already present on machine
> 2021-07-07 01:42:27,861 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Created container zookeeper
> 2021-07-07 01:42:27,863 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Started container zookeeper
> 2021-07-07 01:42:27,865 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Stopping container manager
> 2021-07-07 01:42:27,867 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Stopping container zookeeper
> 2021-07-07 01:42:27,868 [INFO][cluster:785]: [gs-graphmanager-wskhmk-5fc4dc77d-q266l]: Exec lifecycle hook ([kill -TERM `lsof -i:8080 -t`]) for Container "manager" in Pod "gs-graphmanager-wskhmk-5fc4dc77d-q266l_gs-njhjsl(c1f81686-9576-4fee-9065-0ce3a15d8195)" failed - error: command 'kill -TERM `lsof -i:8080 -t`' exited with 1: kill: cannot find process "`lsof -i:8080 -t`"
> , message: "kill: cannot find process \"`lsof -i:8080 -t`\"\n"
> Traceback (most recent call last):
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 563, in start
>     self._waiting_for_services_ready()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 490, in _waiting_for_services_ready
>     if wait_for_deployment_complete(
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/utils.py", line 121, in wait_for_deployment_complete
>     raise K8sError("Deployment {} start failed.".format(name))
> graphscope.framework.errors.K8sError: 'Deployment coordinator-wskhmk start failed.'
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/utils.py", line 156, in wrapper
>     return_value = func(*args, **kwargs)
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/session.py", line 615, in __init__
>     self._connect()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/client/session.py", line 885, in _connect
>     self._launcher.start()
>   File "/opt/anaconda3/envs/DKB_env/lib/python3.8/site-packages/graphscope/deploy/kubernetes/cluster.py", line 573, in start
>     raise K8sError(
> graphscope.framework.errors.K8sError: 'Error when launching Coordinator on kubernetes cluster'

**Reply by lidongze0629** [reply]
Hi, @DanielaCasilli, I can't reproduce the problem in my docker Desktop Kubernetes cluster, Here are the version I used.

Docker Version: 3.4.0 (65384)
Kubernetes version: v1.21.1

Can you upgrade your docker desktop version to the latest and try it again ? Or you can try GraphScope locally without Kubernetes cluster, Thanks.

```python
In [25]: sess = graphscope.session(k8s_volumes=k8s_volumes, k8s_service_type='LoadBalancer')
2021-07-07 14:31:24,811 [INFO][session:581]: Initializing graphscope session with parameters: {'addr': None, 'mode': 'eager', 'cluster_type': 'k8s', 'num_workers': 2, 'preemptive': True, 'k8s_namespace': None, 'k8s_service_type': 'LoadBalancer', 'k8s_gs_image': 'registry.cn-hongkong
.aliyuncs.com/graphscope/graphscope:0.5.0', 'k8s_etcd_image': 'quay.io/coreos/etcd:v3.4.13', 'k8s_image_pull_policy': 'IfNotPresent', 'k8s_image_pull_secrets': [], 'k8s_gie_graph_manager_image': 'registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0', 'k8s_z
ookeeper_image': 'zookeeper:3.4.14', 'k8s_coordinator_cpu': 0.5, 'k8s_coordinator_mem': '512Mi', 'k8s_etcd_num_pods': 1, 'k8s_etcd_cpu': 1.0, 'k8s_etcd_mem': '512Mi', 'k8s_zookeeper_cpu': 0.2, 'k8s_zookeeper_mem': '512Mi', 'k8s_gie_graph_manager_cpu': 0.2, 'k8s_gie_graph_manager_mem
': '512Mi', 'k8s_vineyard_daemonset': 'none', 'k8s_vineyard_cpu': 0.2, 'k8s_vineyard_mem': '512Mi', 'vineyard_shared_mem': '4Gi', 'k8s_engine_cpu': 0.2, 'k8s_engine_mem': '1Gi', 'k8s_mars_worker_cpu': 0.2, 'k8s_mars_worker_mem': '512Mi', 'k8s_mars_scheduler_cpu': 0.2, 'k8s_mars_sche
duler_mem': '512Mi', 'with_mars': False, 'k8s_volumes': {'data': {'type': 'hostPath', 'field': {'path': '/Users/lidongze/alibaba/dataset/ogbn_mag_small', 'type': 'Directory'}, 'mounts': {'mountPath': '/testingdata'}}}, 'k8s_waiting_for_delete': False, 'timeout_seconds': 600, 'dangli
ng_timeout_seconds': 600, 'k8s_client_config': {}}
2021-07-07 14:31:24,919 [INFO][cluster:308]: Launching coordinator...
2021-07-07 14:31:28,053 [INFO][utils:167]: coordinator-uqmctx-75c94bbfcb-rppph: Successfully assigned gs-bghlcn/coordinator-uqmctx-75c94bbfcb-rppph to docker-desktop
2021-07-07 14:31:30,120 [INFO][utils:167]: coordinator-uqmctx-75c94bbfcb-rppph: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0" already present on machine
2021-07-07 14:31:30,121 [INFO][utils:167]: coordinator-uqmctx-75c94bbfcb-rppph: Created container coordinator
2021-07-07 14:31:30,121 [INFO][utils:167]: coordinator-uqmctx-75c94bbfcb-rppph: Started container coordinator
2021-07-07 14:31:37,795 [INFO][cluster:684]: Launching GIE graph manager ...
2021-07-07 14:31:38,905 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Successfully assigned gs-bghlcn/gs-graphmanager-uqmctx-859c6d4bbf-pbb4j to docker-desktop
2021-07-07 14:31:42,352 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:0.5.0" already present on machine
2021-07-07 14:31:42,419 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Created container manager
2021-07-07 14:31:42,599 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Started container manager
2021-07-07 14:31:42,602 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Container image "zookeeper:3.4.14" already present on machine
2021-07-07 14:31:42,678 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Created container zookeeper
2021-07-07 14:31:42,923 [INFO][cluster:785]: [gs-graphmanager-uqmctx-859c6d4bbf-pbb4j]: Started container zookeeper
2021-07-07 14:31:44,954 [INFO][cluster:797]: GIE graph manager service is ready.
2021-07-07 14:31:44,955 [INFO][cluster:541]: Launching etcd ...
2021-07-07 14:31:46,493 [INFO][utils:167]: coordinator-uqmctx-75c94bbfcb-rppph: Readiness probe failed: dial tcp 10.1.0.15:59185: connect: connection refused
2021-07-07 14:31:46,074 [INFO][cluster:807]: Etcd is ready, endpoint is 10.110.189.61:58749
2021-07-07 14:31:46,075 [INFO][cluster:431]: Launching GraphScope engines pod ...
2021-07-07 14:31:46,879 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Successfully assigned gs-bghlcn/gs-engine-uqmctx-ls44c to docker-desktop
2021-07-07 14:31:47,882 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Successfully assigned gs-bghlcn/gs-engine-uqmctx-sksf9 to docker-desktop
2021-07-07 14:31:57,428 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0" already present on machine
2021-07-07 14:31:59,773 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Container image "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.5.0" already present on machine
2021-07-07 14:31:59,774 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Created container engine
2021-07-07 14:31:59,776 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Started container engine
2021-07-07 14:31:59,778 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Created container vineyard
2021-07-07 14:31:59,779 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Started container vineyard
2021-07-07 14:32:00,775 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Created container engine
2021-07-07 14:32:00,776 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Started container engine
2021-07-07 14:32:00,778 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Created container vineyard
2021-07-07 14:32:00,779 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Started container vineyard
2021-07-07 14:32:08,238 [INFO][cluster:864]: [gs-engine-uqmctx-ls44c]: Readiness probe failed:
2021-07-07 14:32:09,168 [INFO][cluster:864]: [gs-engine-uqmctx-sksf9]: Readiness probe failed:
2021-07-07 14:32:17,266 [INFO][cluster:900]: GraphScope engines pod is ready.
2021-07-07 14:32:17,271 [INFO][cluster:1043]: Engines pod name list: ['gs-engine-uqmctx-ls44c', 'gs-engine-uqmctx-sksf9']
2021-07-07 14:32:17,271 [INFO][cluster:1044]: Engines pod ip list: ['10.1.0.17', '10.1.0.19']
2021-07-07 14:32:17,271 [INFO][cluster:1045]: Engines pod host ip list: ['192.168.65.4', '192.168.65.4']
2021-07-07 14:32:17,271 [INFO][cluster:1047]: Vineyard service endpoint: localhost:9600
2021-07-07 14:32:17,271 [INFO][cluster:936]: Starting GAE rpc service on 10.1.0.17:56731 ...
2021-07-07 14:32:20,115 [INFO][coordinator:1131]: Coordinator server listen at 0.0.0.0:59185
2021-07-07 14:32:27,630 [INFO][cluster:565]: Coordinator pod start successful with address localhost:59185, connecting to service ...
2021-07-07 14:33:15,934 [INFO][rpc:111]: GraphScope coordinator service connected.
```

**Reply by sighingnow** [reply]
@DanielaCasilli 

For how to run graphscope on Mac, please refer to https://graphscope.io/docs/deployment.html#deployment-on-local.
